packages/http-server-javascript/lib/main.tsp

```diff
@@ -0,0 +1,4 @@
+// Copyright (c) Microsoft Corporation
```

Comment:
Not sure you need this file if its just an emitter, this basically would just allow import your emitter in the typespec file but do nothing